packages/admin/src/common/toolbar/titleitem/ToolbarTitleItem.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography } from "@material-ui/core";
+import * as React from "react";
+
+import { useStackApi } from "../../../stack";
+import { ToolbarItem } from "../item/ToolbarItem";
+
+interface ToolbarTitleItemProps {
+    title?: React.ReactNode;
```

Comment:
Does it make sense to add a `typographyProps` prop?  

We might not always want `h4` and maybe we want to change some other things... maybe easier than passing in a custom component into `title` with all the stackApi-breadcrumbs-title-thing.

```tsx
interface ToolbarTitleItemProps {
    title?: React.ReactNode;
    typographyProps?: TypographyProps;
}
// ...
const { typographyProps = { variant = "h4" } } = props;
// ...
<Typography {...typographyProps}>
    {stackApi?.breadCrumbs != null &&
        stackApi.breadCrumbs.length > 0 &&
        stackApi.breadCrumbs[stackApi?.breadCrumbs.length - 1].title}
</Typography>
```

Reply:
good idea, i added typographyProps to componentThemeProps of the ToolbarTitleItem